src/test/java/com/commercetools/sync/producttypes/helpers/ProductTypeSyncStatisticsTest.java

```diff
@@ -0,0 +1,27 @@
+package com.commercetools.sync.producttypes.helpers;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ProductTypeSyncStatisticsTest {
+    private ProductTypeSyncStatistics productTypeSyncStatistics;
+
+    @Before
+    public void setup() {
+        productTypeSyncStatistics = new ProductTypeSyncStatistics();
+    }
+
+    @Test
+    public void getReportMessage_WithIncrementedStats_ShouldGetCorrectMessage() {
+        productTypeSyncStatistics.incrementCreated(1);
+        productTypeSyncStatistics.incrementFailed(1);
+        productTypeSyncStatistics.incrementUpdated(1);
```

Comment:
I would give created=1, failed=2, updated=3 so you know that you didn't mix the variables.

Reply:
e93cbf8102c6a2375f8ce8f775afbf9c591f388b